src/install.rst

```diff
@@ -2,30 +2,36 @@
 Installing Nextstrain
 =====================
 
+.. contents::
+   :local:
+
 .. hint::
 
     Before installing, we recommend you read about the :doc:`parts of Nextstrain </learn/parts>`.
 
 The following instructions describe how to install the full suite of Nextstrain's software tools, including:
 
+  * Nextstrain CLI, for access to other tools and management of workflows
   * Augur, for bioinformatic analysis of pathogen genomes
   * Auspice, for interactive visualization of pathogen evolution
-  * Nextstrain CLI, for management of analysis workflows and environments
 
 When completed, you'll be ready to run Nextstrain :term:`workflows <workflow>`.
 
+
```

Comment:
I'd add a link up here (as well as at the bottom of the page) linking out to "Manage installation" -- the two pages are somewhat separated in the sidebar and I presume many people would come to this page first.

Reply:
Thanks!

> I don't love it, but the placement of the "Manage installation" page in the sidebar is the best I could think of. Let me know if you have a better suggestion.

I don't think there's a perfect way to order the sidebar. At least, I've never found one.